src/go/cmd/update-collection-v3/migrations/logs/valuesv2.go

```diff
@@ -0,0 +1,14 @@
+package logs
+
+type ValuesV2 struct {
```

Comment:
It's misleading to name it "v2" because this is not really the "v2" version of values - it's something in between v2 and v3. How about something like "before" or "input"?

Reply:
It is the v2 version of values, although not completely specified. I don't mind changing the name, but at this point the `ValuesV2` and `ValuesV3` naming is consistent across all the migration, and should be changed everywhere if that's what we decide to do.

Reply:
It is not (necessarily) v2, because it has been (or might have been) already modified be the migrations that ran before this one.

In the test, you named it `inputYaml` and `outputYaml` https://github.com/SumoLogic/sumologic-kubernetes-tools/pull/387/files#diff-e5cedc11841a0aac7720f23adbebf74e5072278024ed849c1318337bb448061cR11-R12 - I think there are better names than v2 and v3.